plugin.xml

```diff
@@ -44,7 +44,7 @@
         <!-- The app is able to run in background through audio mode -->
```

Comment:
Comment should be updated to reflect the change:
<!-- The app is able to run in background through remote-notification mode -->